mender-deb-package

```diff
@@ -122,7 +127,9 @@ prepare_recipe() {
     cp support/mender-monitor.service debian/
   fi
 
-  dch --create \
+
+  timestamp="`date +'%a %b %d 12:12:00 %Z %Y'`"
+  faketime "${timestamp}" dch --create \
```

Comment:
`debian/changelog` contains also date, for the same reasons as above: https://github.com/mendersoftware/mender-dist-packages/pull/206/commits/628d55388454e484367d9fe5567c6efe8327228e#r810799323 we cannot generate a changelog entry with the current timestamp.